apple/internal/partials/debug_symbols.bzl

```diff
@@ -101,21 +105,41 @@ def _bundle_dsym_files(ctx, debug_provider, bundle_name, bundle_extension = ""):
     bundle_name_with_extension = bundle_name + bundle_extension
     dsym_bundle_name = bundle_name_with_extension + ".dSYM"
 
-    outputs = []
+    output_binary = ctx.actions.declare_file(
+        "%s/Contents/Resources/DWARF/%s" % (
+            dsym_bundle_name,
+            bundle_name,
+        ),
+    )
+    outputs = [output_binary]
+
+    inputs = []
+    args = [
```

Comment:
nit: use a `ctx.actions.args()`, then you don't need to expand the file paths during analysis